apps/chat-android/README.md

```diff
@@ -0,0 +1,36 @@
+# Amazon Chime SDK Android Chat Demo
+
+## Overview
+
+By completing the steps below, you will learn how to run the Amazon Chime SDK Android chat demo applications on your mobile device or on a simulator. This application enables users to send and receive messages/image attachments in chat channels in real time. This application is designed in a way that can be used as a starting point for your own application, or as a quick way to explore the features of the Amazon Chime SDK messaging. Please see the [Build chat applications in iOS and Android with Amazon Chime SDK messaging](https://aws.amazon.com/blogs/business-productivity/build-chat-applications-in-ios-and-android-with-amazon-chime-sdk-messaging) for more information.
+
+<p align="center">
+<img src="./media/awsChimeSDKMessagingDemo.png" alt="image" width="80%"/>
```

Comment:
Nit: Some texts in the image are blocked by arrows / lines, readability is a little compromised. Do you think we can adjust?

Reply:
Updated.